client/src/pages/Deck/Deck.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom'
+import "./Deck.css";
+import axios from 'axios';
+
+
+// create a page to view deck one cards, and create a get request to our DB
+// upon component did mount GET all cards in deck 1
+class Deck extends Component {
+  constructor(props) {
+      super(props);
+      this.state = {
+          deck: [{_id:123123123, term: 'name', definition: 'george', deck: '1'},{_id:123123123, term: 'name2', definition: 'kevin', deck: 1},]
+      }
+  }
+     
+  // componentDidMount() {
+  //   const {deckNumber} = this.props.location.state
+
+  //   axios.get(`/deck/${deckNumber}`)
+  //     .then((res) => {
+  //       console.log("res: ", res);
+  //       this.setState({deck: res})
+  //     })
+  //     .catch((err) => {
+  //       console.log("err: ", err);
+  //   })
```

Comment:
Please edit your code to use a get request rather than a tester.

Reply:
The code has been updated to reflect a get request to our database as opposed to a test request.